qutebrowser/components/braveadblock.py

```diff
@@ -0,0 +1,325 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2020 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Functions related to ad blocking."""
+
+import io
+import os.path
+import functools
+import logging
+import typing
+import pathlib
+
+from PyQt5.QtCore import QUrl
+
+from qutebrowser.api import (
+    cmdutils,
+    hook,
+    config,
+    message,
+    downloads,
+    interceptor,
+    apitypes,
+    qtutils,
+)
+from qutebrowser.api.interceptor import ResourceType
+
+
+logger = logging.getLogger("misc")
```

Comment:
```suggestion
logger = logging.getLogger("network")
```

for consistency with fe9bde1b1d737a0aa9cbc9ad4346e9a491e31a42.